test/hello_world_dedicated.yaml

```diff
@@ -0,0 +1,30 @@
+---
+environment: "prod"
```

Comment:
I think you missed adding test code for this?

Reply:
I have to create a suite of tests for the CLI. I don't think it makes much sense to add any unit tests for this type of application.

Reply:
Mm I meant you are adding a new test yaml file but it is not being referenced in code, so is this being picked up from some directory directly?

Reply:
oh true, let me add that